src/cryptography/hazmat/backends/openssl/ec.py

```diff
@@ -200,6 +200,23 @@ def private_numbers(self):
             public_numbers=self.public_key().public_numbers()
         )
 
+    def private_bytes(self, encoding, format, encryption_algorithm):
+        evp_pkey = self._backend._lib.EVP_PKEY_new()
+        assert evp_pkey != self._backend._ffi.NULL
+        evp_pkey = self._backend._ffi.gc(
+            evp_pkey, self._backend._lib.EVP_PKEY_free
+        )
+        res = self._backend._lib.EVP_PKEY_set1_EC_KEY(evp_pkey, self._ec_key)
```

Comment:
I found that the EVP APIs get in the way and are often less obvious than the low level APIs for the kind of stuff we do but maybe I was insufficiently good at OpenSSL back then.